pyecharts/engine.py

```diff
@@ -159,40 +159,53 @@ def echarts_js_content_wrap(env, *charts):
     return generate_js_content(*charts)
 
 
-class EchartsEnvironment(Environment):
+# Public API,see document for more detail.
+
+ECHAERTS_TEMPLATE_FUNCTIONS = {
+    'echarts_js_dependencies': echarts_js_dependencies,
+    'echarts_js_dependencies_embed': echarts_js_dependencies_embed,
+    'echarts_container': echarts_container,
+    'echarts_js_content': echarts_js_content,
+    'echarts_js_content_wrap': echarts_js_content_wrap
+}
+
+
+class PyEchartsConfigMixin(object):
```

Comment:
I understand that this class defines the common interface but too few public methods to be a class and you have to spend extra documentation to explain why PyEchartsConfigMixin should be there. So, I would recommend to get rid of it.